MediaBrowser.Providers/Manager/ItemImageProvider.cs

```diff
@@ -273,7 +273,7 @@ public bool ValidateImages(BaseItem item, IEnumerable<IImageProvider> providers,
                 }
 
                 if (!refreshOptions.ReplaceAllImages &&
-                    refreshOptions.ReplaceImages.Length == 0 &&
+                    !refreshOptions.ReplaceImages.Any() &&
```

Comment:
You should be able to check for `Count == 0` here instead of `Any()`

Reply:
Some linter suggestion came up that using Any could be faster since it doesn't have to count all items, just figure if it is empty or not. But I'm happy to do it either way.

Reply:
I guess it depends on if it's `Count()` or `Count`. If the latter isn't available `Any()` is better.